LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2007 Weifen Luo (email: weifenluo@yahoo.com)
```

Comment:
OK only the WinFormsUI is (C) Weifen Luo.
But the whole rest of RosDBG, according to the list of commits, should be (C) 2008-2011 Art Yerkes (@prozacchiwawa), (C) 2008-2011 Christopher von Wittich (@christophvw), (C) 2008-2011 Ged Murphy (@gedmurphy), (C) 2008-2011 Gregor Schneider (@grschneider), and other contributors.
Please check with the authors on Mattermost for example.

Reply:
Yeah, my mistake

Reply:
https://github.com/reactos/reactosdbg/pull/2